app.py

```diff
@@ -51,11 +52,13 @@ def _get_rv_binary(self):
         # get the setting
         system = sys.platform
```

Comment:
Need this still? At a glance I don't think `system` is used anymore.

Reply:
ah good catch!